resspect/lightcurves_utils.py

```diff
@@ -0,0 +1,160 @@
+"""
+    utils for fit_lightcurves methods
+"""
+
+import io
+import tarfile
+from typing import AnyStr
+from typing import Tuple
+
+import numpy as np
+import pandas as pd
+
+from resspect.snana_fits_to_pd import read_fits
+
+SNPCC_LC_MAPPINGS = {
+    "snii": ['2', '3', '4', '12', '15', '17', '19', '20', '21', '24', '25',
+             '26', '27', '30', '31', '32', '33', '34', '35', '36', '37', '38',
+             '39', '40', '41', '42', '43', '44'],
+    "snibc": ['1', '5', '6', '7', '8', '9', '10', '11', '13', '14', '16',
+              '18', '22', '23', '29', '45', '28']
+}
+
+
+def read_file(file_path: str) -> list:
+    with open(file_path, "r") as file:
```

Comment:
you are shadowing `file` here.  It doesn't cause any actual problems here, but I still wouldn't do it.

Reply:
I think it looks fine?

Reply:
you're right.  It's only an issue in Python2.  I'm not sure if this code is supposed to be Python3-only?  It's not a big deal but I wouldn't use `file` just to be safe.

Reply:
Right, this code supposed to be Python>=3.7. I changed `file` to `input_file`